schema-engine/sql-schema-describer/src/mssql.rs

```diff
@@ -61,7 +61,7 @@ static DEFAULT_DB_GEN: Lazy<Regex> = Lazy::new(|| Regex::new(r"\((.*)\)").unwrap
 /// ```ignore
 /// CREATE DEFAULT catcat AS 'musti';
 /// ```
-static DEFAULT_SHARED_CONSTRAINT: Lazy<Regex> = Lazy::new(|| Regex::new(r"^CREATE DEFAULT (.*)").unwrap());
```

Comment:
so odd, I wonder why we had this constraint of it being at the start in the first place

Reply:
I actually thing that is a pretty fair assumption to make, who knew that comments or special characters would be stored there as well.